src/content/en/updates/posts/2015/10/chrome-47-webrtc.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: updates/post
+title: "Chrome 47 WebRTC: media recording, secure origins &amp; proxy&nbsp;handling"
+description: "Chrome 47 has several significant WebRTC enhancements and updates including audio and video recording, proxy handling and mandatory secure origins for getUserMedia()."
+published_on: 2015-10-28
+updated_on: 2015-10-28
+authors:
+  - samdutton
+tags:
+  - news
+  - audio
+  - video
+  - media
+  - WebRTC
+  - getUserMedia
+featured_image: /web/updates/images/2015-10-28-chrome-47-webrtc/featured.png
+---
+
+<p class="intro">Chrome 47 includes several significant WebRTC enhancements and updates.</p>
+
+### Record video from your web apps
+
+The `MediaStreamRecorder` API has long been the top chromium.org request, with over 2500
+stars. Media recording has now been added to Chrome behind the experimental Web Platform features flag &mdash; though it's desktop only (and video-only) for the moment. This allows you to record and play back or download video. There is a simple demo [here](https://rawgit.com/Miguelao/demos/master/mediarecorder.html) and you can find out more from the [discuss-webrtc announcement](https://groups.google.com/forum/?#!msg/discuss-webrtc/n11m846oV4I/0b3ycjmjCAAJ).
+
+### Audio output device selection
+
+`MediaDevices.enumerateDevices()` has been released: more details details in [Issue
```

Comment:
Period after released
New sentence
Add verb to new sentence
eliminate duplicate word.